.editorconfig

```diff
@@ -0,0 +1,2 @@
+[*.kt]
+indent_size = 2
```

Comment:
is this really really really necessary?

Reply:
Maybe not, but i got it into my head this was the improved way of specifying indent_size here for editors of all shapes and sizes and for various kotlin plugins.

Hence removal of 

```
  spotless {
    kotlin {
      target "**/*.kt"
      ktlint(dep.ktlintVersion).userData(['indent_size': '2', 'continuation_indent_size': '2'])
    }
  }
```

Do we have a good working project to copy the setup of verify stage plugins and code formatters?

Reply:
Also Intellij seems headed this way

https://www.jetbrains.com/help/idea/configuring-code-style.html#editorconfig

Reply:
Works for me! Yes, let's do this universally then.